dotcom-rendering/src/web/components/TextBlockComponent.tsx

```diff
@@ -227,6 +227,16 @@ export const TextBlockComponent = ({
 						prefix: '<h3>',
 						suffix: '</h3>',
 					},
+					{
+						prefix: '<footer>',
+						suffix: '</footer>',
+						unwrappedElement: 'footer',
+					},
+					{
+						prefix: '<ol>',
+						suffix: '</ol>',
+						unwrappedElement: 'ol',
```

Comment:
I have indeed found Kibana logs for these two html components that are added:
* [`ol` log](https://logs.gutools.co.uk/s/dotcom/app/discover#/doc/50201630-bd0f-11e9-92b3-f10623802d36/logstash-frontend-2023.03.06?id=lPMcuIYBBLa5BHv4GDBs)
* [`footer` log](https://logs.gutools.co.uk/s/dotcom/app/discover#/doc/50201630-bd0f-11e9-92b3-f10623802d36/logstash-frontend-2023.03.06?id=3MMcuIYBgN56anDjGCEQ)

Assuming from Kibana we can verify that these are the only two components that need to be added?